common/containers/TabSection/index.tsx

```diff
@@ -27,9 +24,10 @@ interface Props {
   changeNode: TChangeNode;
   changeGasPrice: TChangeGasPrice;
 }
-class App extends Component<Props, {}> {
+class TabSection extends Component<Props, {}> {
   public render() {
     const {
+      location,
```

Comment:
Gonna pull this down to confirm, but I don't still don't see how TabSection is getting location passed to it